### PR TITLE
Adds An Alternate High Visibility Jacket

### PR DESCRIPTION
**Pull request by KingOfThePing**
To feel useful again and to procrastinate on a million other things this PR adds a high visibility jacket:

It can be toggled open and closed and has two pocket slots. It covers the upper + lower torso as well as the arms.
Functionality has been tested locally and it works without issue.

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| \icons\obj\clothing\suits.dmi (jacket_hazard)| Unknown (Shiptest) | CC0 |

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
contained sprite, please.

**Comment by KingOfThePing** [comment]
Why? It's just another jacket without any special functionality

**Comment by SleepyGemmy** [comment]
> Why? It's just another jacket without any special functionality

because contained sprites is the praxis.

**Comment by KingOfThePing** [comment]
- re-shuffled the sprites into a contained sprite
- made it an alternate version of the existing high vis jacket
- added it to the loadout jacket selection

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -304,7 +304,6 @@
 	icon = 'icons/contained_items/clothing/topwear/bridge_crew_jacket_sancol.dmi'
 	icon_state = "bridge_crew_jacket_sancol"
 	item_state = "bridge_crew_jacket_sancol"
-
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/miscellaneous.dm:307]: unintentional (?) newline removal.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -387,7 +386,14 @@
 	item_state = "highvis"
 	body_parts_covered = UPPER_TORSO|ARMS
 	contained_sprite = TRUE
-
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/miscellaneous.dm:390]: unintentional (?) newline removal.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -387,7 +386,14 @@
...
+/obj/item/clothing/suit/storage/toggle/highvis_alt
+	name = "high visibility jacket"
+	desc = "A bright yellow jacket with reflective stripes. For use in operations, engineering and sometimes even law enforcement in cold and poor weather or when visibility is low."
+	icon = 'icons/clothing/suits/highvis.dmi'
+	icon_state = "highvis_alt"
+	item_state = "highvis_alt"
+	body_parts_covered = UPPER_TORSO|ARMS
+	contained_sprite = TRUE
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/miscellaneous.dm:396]: newline below this line.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -387,7 +386,14 @@
 	item_state = "highvis"
 	body_parts_covered = UPPER_TORSO|ARMS
 	contained_sprite = TRUE
-
+/obj/item/clothing/suit/storage/toggle/highvis_alt
+	name = "high visibility jacket"
+	desc = "A bright yellow jacket with reflective stripes. For use in operations, engineering and sometimes even law enforcement in cold and poor weather or when visibility is low."
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/miscellaneous.dm:391]:
```suggestion
	desc = "A bright yellow jacket with reflective stripes. For use in operations, engineering, and sometimes even law enforcement in cold and poor weather or when visibility is low."
```

**Review by SleepyGemmy — APPROVED** [review]
requested changes addressed. PR looks fine.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**